README.md

```diff
@@ -324,6 +324,13 @@ Emitted when a LevelDB query is made using the `leveldown` module.
     * `value` (Object) The value being added to the LevelDB database using the `put` method (Undefined for other methods) 
     * `opCount` (Number) The number of operations carried out by a `batch` method (Undefined for other methods) 
     * `duration` (Number) the time taken for the LevelDB query to be responded to in ms.
+    
+### Event: 'loopback-datasource-juggler'
+Emitted when a function is called on the `loopback-datasource-juggler` module
```

Comment:
https://github.com/sjanuary/appmetrics/blob/5da4bafe2564dadd33b39f8937fd866356fc5ea0/probes/loopback-probe.js#L56-L57 shows the functions being intercepted and it looks like they're on `DataAccessObject`. Ideally the person who wrote the probe should have done the docs at the same time because he's not around any more and I'm trying to work out from the code what it does... I'm happy to make changes though, e.g. could list the functions in the description if that would be helpful.